samples/mcuboot_config/mcuboot_config.template.h

```diff
@@ -146,4 +146,15 @@
  *    do { do watchdog feeding here! } while (0)
  */
 
+/*
+ * If a OS ports support single thread mode or is bare-metal then:
+ * This macro shall implements call which switches CPU to idle states
+ * which wakes on an event like expected uart transmission.
```

Comment:
this macro implements call that switches CPU to an idle state, from which the CPU may be woken up by, for example, UART transmission event.

Reply:
thanks!